python/src/tokenize.rs

```diff
@@ -7,9 +7,10 @@
 use pyo3::prelude::*;
 use pyo3::types::PyList;
 
+use crate::utils::{deserialize_params, serialize_params};
 use vtext::tokenize::*;
 
-#[pyclass]
+#[pyclass(module = "vtext.tokenize")]
```

Comment:
Due to https://github.com/PyO3/pyo3/issues/474 that should have been fixed, but I still see the same error.